### DISCUSSION TITLE
AutoFac 7.0

### DISCUSSION THREAD

**Discussion by TonyValenti** [abpframework/abp#15941] [Ideas] [answered]
Hi All,
AutoFac released version 7.0 which can now inject to required properties. 

Please evaluate this for the next major release of ABP. This can eliminate TONS of boilerplate constructor code, especially when inheritance is involved.

**Comment by maliming** [comment, marked as answer]
hi

I create an issue. https://github.com/abpframework/abp/issues/15942
